addon/components/draggable-object.js

```diff
@@ -80,7 +75,7 @@ export default Ember.Component.extend({
       event.preventDefault();
       return;
     } else {
-      Ember.run.later(()=> {
+      Ember.run.next(()=> {
```

Comment:
`run.later` expects 2 params: the callback and the time delay.  Without setting a time for the function to wait, it is essentially a `next`.